server/models/groups.js

```diff
@@ -0,0 +1,57 @@
+const { verifyAccessToken } = require('../utils/token-helpers');
+const { HttpError } = require('../utils/errors');
+const prisma = require('../db');
+
+class Group {
+  constructor(name, token) {
+    this.name = name;
+    this.token = token;
+  }
+
+  async groupByUserExists() {
```

Comment:
```suggestion
  async isDuplicate() { 
```

This might be a clearer function name

Reply:
Done, thanks for the suggestion it is better